### DISCUSSION TITLE
Welcome to flutter_form_builder Discussions!

### DISCUSSION THREAD

**Discussion by danvick** [danvick/flutter_form_builder#628] [General]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by WilliamCunhaCardoso** [comment]
Hello there, my name is William Cardoso. I am a contributor (and now a collaborator). I often submit fixes, enhancements and answer some issues.
